MatrixKit/Models/Account/MXKAccountManager.m

```diff
@@ -344,7 +353,17 @@ - (BOOL)isAPNSAvailable
 
 - (void)loadAccounts
 {
-    NSData *accountData = [[NSUserDefaults standardUserDefaults] objectForKey:@"accounts"];
+    NSUserDefaults *sharedDefaults = [MXKAppSettings standardAppSettings].sharedUserDefaults;
+    
+    NSData *oldAccountData = [[NSUserDefaults standardUserDefaults] objectForKey:kMXKAccountsKey];
```

Comment:
For code maintenance, please a comment explaining that we are a migration of data.